.github/workflows/check.yml

```diff
@@ -95,6 +95,26 @@ jobs:
       - x64
       - sre
     steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
```

Comment:
We've added a toolchain.toml, which pins the rust toolchain we're using. actions-rs does not support that yet, so you'll have to manually install rustup + cargo.